pycrostates/preprocessing/extract_gfp_peaks.py

```diff
@@ -70,23 +86,39 @@ def extract_gfp_peaks(
 
     # retrieve picks
     picks = _picks_to_idx(inst.info, picks, none="all", exclude="bads")
-
+    picks_all = _picks_to_idx(
+        inst.info, inst.ch_names, none="all", exclude="bads"
+    )
+    _check_picks_uniqueness(inst.info, picks)
     # retrieve data array
     kwargs = (
         dict()
         if isinstance(inst, BaseEpochs)
         else dict(reject_by_annotation=reject_by_annotation)
     )
     data = inst.get_data(picks=picks, tmin=tmin, tmax=tmax, **kwargs)
+    if return_all:
+        data_all = inst.get_data(
+            picks=picks_all, tmin=tmin, tmax=tmax, **kwargs
+        )
```

Comment:
```suggestion
    data = inst.get_data(picks=picks_all if return_all else picks, tmin=tmin, tmax=tmax, **kwargs)
```

let's call get_data only once